Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -1173,22 +1173,29 @@ RefPtr<WebCore::ScrollingCoordinator> WebChromeClient::createScrollingCoordinato
 #endif
 
 #if PLATFORM(MAC)
-std::unique_ptr<ScrollbarsController> WebChromeClient::createScrollbarsController(Page& corePage, ScrollableArea& area) const
+void WebChromeClient::ensureScrollbarsController(Page& corePage, ScrollableArea& area) const
 {
     auto page = protectedPage();
     ASSERT(page->corePage() == &corePage);
-    
+    auto* currentScrollbarsController = area.existingScrollbarsController();
     if (area.mockScrollbarsControllerEnabled())
```

Comment:
Should this assert that we've already created a mock scrollbars controller, or does that happen later?

Reply:
Added assert(). We should never get into this code if mock scrollbars are enabled but adding it as a sanity check is fine.